policyengine_us/variables/household/expense/health/employer_premium_contribution.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class EmployerPremiumContribution(Enum):
+    NONE = "NONE"
```

Comment:
And so on for other values.
```suggestion
    NONE = "NONE" # Employer paid none of premiums.
```